src/spikeinterface/postprocessing/template_metrics.py

```diff
@@ -64,6 +64,8 @@ class ComputeTemplateMetrics(AnalyzerExtension):
         For more on generating a ChannelSparsity, see the `~spikeinterface.compute_sparsity()` function.
     include_multi_channel_metrics : bool, default: False
         Whether to compute multi-channel metrics
+    delete_existing_metrics : bool, default: False
+        If True, deletes any quality_metrics attached to the `sorting_analyzer`
```

Comment:
Could this state what happens when `False`? (i.e. previously computed are propagated, but will be deleted if kwargs don't match).